bedrock/firefox/templates/firefox/dev-firstrun.html

```diff
@@ -220,17 +215,6 @@
           </div>
         </li>
         <li class="feature">
-          <a href="https://www.youtube.com/watch?v=1R9_WdXwUsE" class="video-play" rel="external">
-            <img src="{{ static('img/firefox/developer/feature-webide.jpg') }}" alt="{{ _('Screenshot') }}" class="screenshot">
-          </a>
-          <h2>{{_('WebIDE')}}</h2>
-          <p>{{_('Develop, deploy and debug Firefox OS apps directly in your browser, or on a Firefox OS device, with this tool that replaces App Manager.')}}</p>
-          <a href="https://developer.mozilla.org/docs/Tools/WebIDE" class="more" rel="external">{{_('Learn more about WebIDE')}}</a>
-          <div class="responsive-video-container">
-            <div class="video" data-video-id="1R9_WdXwUsE"></div>
-          </div>
-        </li>
```

Comment:
Have we removed the WebIDE from dev edition or do we simply not want to call it out anymore as it will be removed?